Makefile

```diff
@@ -78,7 +78,8 @@ all: lint test manager
 
 # Run tests
 native-test:
-	GO111MODULE=on go test -mod vendor ./pkg/... ./apis/... -coverprofile cover.out
+	GO111MODULE=on go test -mod vendor ./pkg/controller/constrainttemplate -coverprofile cover.out
```

Comment:
Was this temporarily modified to only test CTs? It looks like this should be switched back.

Reply:
I keep forgetting things like this 😵‍💫.  Thanks for catching!